src/_configure_plextrac.sh

```diff
@@ -49,9 +49,12 @@ RUNAS_APPUSER=True
 PLEXTRAC_PARSER_URL=https://plextracparser:4443
 UPGRADE_STRATEGY=${UPGRADE_STRATEGY:-"stable"}
 PLEXTRAC_BACKUP_PATH="${PLEXTRAC_BACKUP_PATH:-$PLEXTRAC_HOME/backups}"
+CKEDITOR_ENVIRONMENT_SECRET_KEY=${CKEDITOR_ENVIRONMENT_SECRET_KEY:-`generateSecret`}
+CKEDITOR_MIGRATE=true
```

Comment:
@mmorgan-pt If this goes to production, `CKEDITOR_MIGRATE` should be false for now. We'll want to set this variable true for select customers as we are ready.